DogeNetWebClient/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Routes, RouterModule } from '@angular/router';
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { ChatContainerComponent } from './messenger/pages/chat-container/chat-container.component';
+import { NotFoundComponent } from './shared/components/not-found/not-found.component';
+import { ChatBoxComponent } from './messenger/pages/chat-box/chat-box.component';
+
+const routes: Routes = [
+  { path: 'messages', component: ChatContainerComponent },
```

Comment:
Maybe it is better to create separate module for chat and place there another routing module?